assets/css/NewEvaluation.less

```diff
@@ -0,0 +1,148 @@
+body {
+box-sizing: border-box;
+background-color: #f7f7f7;
+
+}
+
+.borderSwitch{
+border-bottom: solid thick #03a9f4;
```

Comment:
Define all colors in a colors.less file and use the vars instead of hardcoding the colors